### PR TITLE
Dm 3720

### PR DESCRIPTION
**Pull request by bradjohnson92008**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-3720

## Description - what does this code do?
This code changes the LDAP port we use to authenticate internal users from port 3269 to LDAP secure port 3269.

## Testing done - how did you test it/steps on how can another person can test it 
Unfortunately you can't test this locally only on CAG.  
The test is just logging in to one of our servers with this code on it and verify that you are logged in correctly as an internal user.  If you see the "Your Profile" link in the upper right corner of the home page your authenticated.

image

## Screenshots, Gifs, Videos from application (if applicable)
image

## Acceptance criteria

- Our LDAP string is making insecure connections
- Determine what is it about our string? What are we doing that’s not secure? 
- Review PDFs team sent 
- Print out and debug out our LDAP string, compare to what they’re recommending
- Could be we’re using the wrong port number
- It’s different to every application how those strings are built?
- Determine next course of action
- We could POAM this
- Done=Get us off the list

## Definition of done
Get us off the List.
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: config/ldap.yml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-development:
-  host: gc.vha.med.va.gov #VHALON3DC4.v22.med.va.gov #VAISH1DC1.va.gov gc.vha.med.va.gov #vac103dc2vha.vha.med.va.gov
-  port: 3268 #389 #3268 global catalog server?
-  default_domain: va.gov
-  base: DC=va,DC=gov
+#development:
+#  host: gc.vha.med.va.gov #VHALON3DC4.v22.med.va.gov #VAISH1DC1.va.gov gc.vha.med.va.gov #vac103dc2vha.vha.med.va.gov
+#  port: 3268 #389 #3268 global catalog server?
+#  default_domain: va.gov
+#  base: DC=va,DC=gov
```
Comment (JoshingYou1) [inline config/ldap.yml:8]: If we're not using this, can we remove it?

File: config/ldap.yml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-test:
-  host: examplehost
-  port: 389
-  default_domain: EXAMPLEDOMAIN
-  base: examplebase # OU=US-Tampa Bay,OU=North America,OU=Accounts,DC=ourdomain,DC=org
+#test:
+#  host: examplehost
+#  port: 389
+#  default_domain: EXAMPLEDOMAIN
+#  base: examplebase # OU=US-Tampa Bay,OU=North America,OU=Accounts,DC=ourdomain,DC=org
```
Comment (JoshingYou1) [inline config/ldap.yml:14]: If we're not using this, can we remove it?

**Review by JoshingYou1 — CHANGES_REQUESTED** [review]
I verified that I got signed in automatically. Nice work Brad! Left a couple of comments for ya.

File: config/ldap.yml (hunk 1 above)
Comment (bradjohnson92008) [reply config/ldap.yml]: Removed it.

File: config/ldap.yml (hunk 2 above)
Comment (bradjohnson92008) [reply config/ldap.yml]: yep removed it.

**Review by JoshingYou1 — APPROVED** [review]
LGTM!

**Review by camillevilla — APPROVED** [review]
👍 Working for me!

**Pull request merged**